recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ hash }}
 
 build:
-  number: 5
+  number: 0
```

Comment:
this doesn't look correct, build should only reset if version changed.

Reply:
so i increment to 6 instead?